src/antares/service/api_services/st_storage_api.py

```diff
@@ -73,3 +74,18 @@ def get_storage_matrix(self, storage: STStorage, ts_name: STStorageMatrixName) -
         except APIError as e:
             raise STStorageMatrixDownloadError(storage.area_id, storage.id, ts_name.value, e.message) from e
         return dataframe
+
+    def read_st_storages(self, area_id: str) -> List[STStorage]:
+        url = f"{self._base_url}/studies/{self.study_id}/areas/{area_id}/storages"
+        json_storage = self._wrapper.get(url).json()
+        storages = []
+
+        for storage in json_storage:
+            storage_id = storage.pop("id")
+            storage_name = storage.pop("name")
+
+            storage_properties = STStorageProperties(**storage)
+            st_storage = STStorage(self.config, storage_id, storage_name, storage_properties)
+            storages.append(st_storage)
+
+        return storages
```

Comment:
I forgot this but it could be nice to sort the objects based on their ids, as you did inside the `read_areas` method to ensure test reproducibility.
**Same goes for the 3 services**